LFX: Add KCL projects for Term 01

zong-zhe opened the pull request:
Hi, team
This pr adds KCL project for LFX term 01 2025, thanks for your reviewing: )

nate-double-u commented:
Thanks for this @zong-zhe & @He1pa!

Usually we like to ask that there are 2 mentors per project -- would that be possible here?

Also, I'm not very familiar with your project (forgive me), is this enough work for a 12 week, full time program?

zong-zhe commented:
> Thanks for this @zong-zhe & @He1pa!
> 
> Usually we like to ask that there are 2 mentors per project -- would that be possible here?
> 
> Also, I'm not very familiar with your project (forgive me), is this enough work for a 12 week, full time program?

Thanks for your suggestion 😄 , I will fix these comments and add some more content

zong-zhe commented:
Hi @nate-double-u @He1pa 😄 

I fixed the issues you mentioned and added some more details. Please help us see what else needs to be improved. Thanks for your reviewing: )

nate-double-u reviewed (changes requested):
Hi @zong-zhe, thanks for making those changes. It looks like your PR has caught some extra information. Could you fix this so it's only adding your own projects proposals?

zong-zhe commented:
> Hi @zong-zhe, thanks for making those changes. It looks like your PR has caught some extra information. Could you fix this so it's only adding your own projects proposals?

Hi @nate-double-u 😄 

I have cleaned up the commits.
